extmod/modussl_axtls.c

```diff
@@ -173,6 +216,22 @@ STATIC mp_uint_t ussl_socket_read(mp_obj_t o_in, void *buf, mp_uint_t size, int
     return size;
 }
 
+STATIC mp_obj_t ussl_socket_recv(mp_obj_t self_in, mp_obj_t len_in) {
+    size_t len = mp_obj_get_int(len_in);
```

Comment:
mp_obj_int_get_uint_checked or similar would be more correct (range checking, less casting, length cannot be negative anyway) and safer (if len_in is effectively negative casting to size_t makes a huge number which is going to crash the application)

Reply:
Thanks for the suggestion! I'm just finding my way through the many, mysterious, undocumented, and hard to find mp_ functions and macros :-). I'll fix.

Reply:
It's not that hard to find, but I admit you gotta find out how: public API is mostly obj.h and runtime.h and the naming is quite consistent so I know for instance there has to be a way to get a uint so it's just ctrl-p `get uint` (or whatever your fuzzy match sohurtcut is, or otherwise use find with regexes)